environments.json

```diff
@@ -2462,14 +2462,41 @@
     "release": "2020-05-21",
     "retire": "2020-07-16",
     "equals": "chrome83",
-    "obsolete": false
+    "obsolete": true
```

Comment:
to be clear, edge 86 and 87 are the ones i'd consider non-obsolete; it's fine to treat edge 83-85 as `true`.